workbench-tests/pom.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <!-- TODO use a different parent -->
+  <parent>
+    <groupId>org.kie.tests</groupId>
+    <artifactId>kie-tests</artifactId>
+    <version>6.3.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>workbench-tests</artifactId>
+  <packaging>pom</packaging>
+  <name>Workbench Tests</name>
+  <description>Workbench tests organized by features/use cases.</description>
+  <modules>
+    <module>workbench-tests-authoring</module>
```

Comment:
Didn't we think that "authoring" was too broad a category? Can we see a way of introducing more granularity here?